framework/src/modules/reward/endpoint.ts

```diff
@@ -43,4 +43,12 @@ export class RewardEndpoint extends BaseEndpoint {
 	public getRewardTokenID(): string {
 		return this._config.tokenID.toString('hex');
 	}
+
+	public getInflationRate(context: ModuleEndpointContext): InflationRate {
+		const reward = BigInt(this.getDefaultRewardAtHeight(context).reward);
+		const blocksPerYear = BigInt(Math.floor((365 * 24 * 60 * 60) / this._config.blockTime));
```

Comment:
blocktime can be obtained from genesis config in the `init` function, so please use that instead of adding the the module config 🙏

Reply:
same should be done for [validators](https://github.com/LiskHQ/lisk-sdk/blob/development/framework/src/modules/validators/module.ts#L66) module then ?

Reply:
ah yes, good catch! it should be taken from the genesis config

Reply:
Validators module related code & tests to be updated under issue #8050